port/common/omrstr.c

```diff
@@ -307,6 +307,13 @@ omrstr_convert(struct OMRPortLibrary *portLibrary, int32_t fromCode, int32_t toC
 			}
 		}
 		break;
+#if defined(OMR_OS_WINDOWS)
+		case J9STR_CODE_WINTHREADACP:
+			/* fall through */
+		case J9STR_CODE_WINDEFAULTACP:
+			result = convertMutf8ToPlatform(portLibrary, (toCode == J9STR_CODE_WINTHREADACP) ? CP_THREAD_ACP : CP_ACP, inBuffer, inBufferSize, outBuffer, outBufferSize);
+			break;
```

Comment:
Since you're already switching on `toCode`, I think this would be better written as:
```c++
		case J9STR_CODE_WINTHREADACP:
			result = convertMutf8ToPlatform(portLibrary, CP_THREAD_ACP, inBuffer, inBufferSize, outBuffer, outBufferSize);
			break;
		case J9STR_CODE_WINDEFAULTACP:
			result = convertMutf8ToPlatform(portLibrary, CP_ACP, inBuffer, inBufferSize, outBuffer, outBufferSize);
			break;
```

Reply:
Sure, updated accordingly. Please have another look.